api/v1alpha1/virtualmachine_types.go

```diff
@@ -215,6 +215,30 @@ type VirtualMachineSpec struct {
 	// responding to the probe.
 	// +optional
 	ReadinessProbe *Probe `json:"readinessProbe,omitempty"`
+
+	// AdvancedOptions describes a set of optional, advanced options for configuring a VirtualMachine
+	AdvancedOptions *VirtualMachineAdvancedOptions `json:"advancedOptions,omitempty"`
+}
+
+// AdvancedOptions describes a set of optional, advanced options for configuring a VirtualMachine
+type VirtualMachineAdvancedOptions struct {
+	// DefaultProvisioningOptions specifies the provisioning type to be used by default for all VirtualMachine volumes.
```

Comment:
Clarification, only volumes (ie. disks) that are exclusively owned by this VM. 
Does not apply to PersistentVolumeClaim volumes that are created and managed externally.